microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -36,6 +40,13 @@ This release adds improvements related to the following components and concepts.
 ** If you run cAdvisor as a standalone DaemonSet for monitoring pods and containers, update it to v0.43.0 or later.
 ** If you deploy Java applications with the JDK, ensure you are using JDK 11.0.16 and later or JDK 15 and later, which fully support cgroup v2.
 
+[id="microshift-4-14-updates-supported"]
+=== Updates are supported on {product-version} and later
```

Comment:
"and later" is pretty important, because it means that upgrades from previous minor version (4.12, V4.13) are not supported. I would prefer we state that explicitly.